.github/workflows/pull_request_verification.yml

```diff
@@ -21,7 +21,13 @@ jobs:
       - name: Grant execute permission for gradlew
         run: chmod +x gradlew
       - name: Check lint
-        run: ./gradlew lintDebug
+        run: |
+          gpg --quiet --batch --yes --decrypt --passphrase="$FIREBASE_SECRET" \
+          --output /home/runner/work/Picover/Picover/app/services.tar **/services.tar.gpg
+          tar xvf /home/runner/work/Picover/Picover/app/services.tar
+          ./gradlew lintDebug
+        env:
+          FIREBASE_SECRET: ${{ secrets.FIREBASE_SECRET }}
```

Comment:
probably other way to handle this, but You probably noticed already :D : https://write.agrevolution.in/ci-android-part-1-964026a0356f